docs/file-reference.md

```diff
@@ -869,6 +884,61 @@ secrets:
 Anything [EnvVarSource Spec](https://kubernetes.io/docs/api-reference/v1.8/#envvarsource-v1-core) from Kubernetes can be included within the Kedge file.
 
 
+## imageStream
+
+```yaml
+imageStreams:
+- <imageStream>
+```
+
+> Example
+
+```yaml
+name: rubyapp
+imageStreams:
+- name: rubystream
+  dockerImageRepository: "docker.io/openshift/ruby-20-centos7"
+```
+
+<aside class="notice">
+Each "imageStream" is an OpenShift <a target="_blank" href="https://docs.openshift.org/latest/rest_api/apis-image.openshift.io/v1.ImageStream.html#object-schema">ImageStream Spec</a> with additional Kedge-specific keys.
+</aside>
+
+
+| Type                         | Required | Description |
+|----------------------------------|--------------|------|
+| name | string   | yes          | The name of the ImageStream |
+
+
+### name
+
+```yaml
+name: wordpress
+```
+
+| Type | Required | Description |
+|----------|--------------|-------|
+| string   | yes          | The name of the ImageStream |
+
+### OpenShift extension
+
+> Example extending `imageStreams` with OpenShift Ingress Spec
+
+```yaml
+name: webapp
+imageStreams:
+- tags:
+  - from:
+      kind: DockerImage
+      name: centos/httpd-24-centos7:latest
+    name: "2.4"
+    referencePolicy:
+      type: Source
```

Comment:
This is default value do we need it here?

Reply:
ack, removed